src/vivarium_gates_iv_iron/components/pregnancy.py

```diff
@@ -40,25 +45,54 @@ def setup(self, builder: Builder):
                                                      key_columns=['sex'],
                                                      parameter_columns=['age', 'year'])
         self.conception_rate = builder.value.register_rate_producer('conception_rate', source=conception_rate)
+        conception_rate_data = conception_rate_data.set_index([col for col in metadata.ARTIFACT_INDEX_COLUMNS if col != "location"])
 
         outcome_probabilities = self.load_pregnancy_outcome_probabilities(builder)
         self.outcome_probabilities = builder.lookup.build_table(outcome_probabilities,
                                                                 key_columns=['sex'],
                                                                 parameter_columns=['age', 'year'])
-        # TODO remove age and sex columns when done debugging
-        self.population_view = builder.population.get_view(self.columns_created + ['age', 'sex'])
+
+        life_expectancy_data = builder.data.load("population.theoretical_minimum_risk_life_expectancy")
+        self.life_expectancy = builder.lookup.build_table(life_expectancy_data, parameter_columns=['age'])
+
+        all_cause_mortality_data = builder.data.load("cause.all_causes.cause_specific_mortality_rate").set_index([col for col in metadata.ARTIFACT_INDEX_COLUMNS if col != "location"])
+        maternal_disorder_csmr = builder.data.load("cause.maternal_disorders.cause_specific_mortality_rate").set_index([col for col in metadata.ARTIFACT_INDEX_COLUMNS if col != "location"])
+        self.background_mortality_rate = builder.lookup.build_table((all_cause_mortality_data - maternal_disorder_csmr).reset_index(),
+                                                                    key_columns=['sex'],
+                                                                    parameter_columns=['age', 'year'])
+        maternal_disorder_incidence = builder.data.load("cause.maternal_disorders.incidence_rate").set_index([col for col in metadata.ARTIFACT_INDEX_COLUMNS if col != "location"])
+
+        self.probability_maternal_deaths = builder.lookup.build_table((maternal_disorder_csmr / conception_rate_data).reset_index(),
```

Comment:
I'm assuming probability refers to the probability of death due to maternal disorders given that you are pregnant. If so, this equation is correct if `conception_rate = conceptions / person_time_total`. Is that what this data represents? It seems to me that a better definition of conception rate would be `conception_rate = conceptions / person_time_not_pregnant`

Reply:
conception rate is the same as incidence of pregnancy, defined here: https://vivarium-research.readthedocs.io/en/latest/other_models/pregnancy/index.html#id7